Objects/Objects/GIS/CRS.cs

```diff
@@ -0,0 +1,14 @@
+using Speckle.Core.Models;
+
+namespace Objects.GIS;
+
+public class CRS : Base
+{
+  public string? name { get; set; }
+  public string? authority_id { get; set; }
+  public string? wkt { get; set; }
+  public string? units_native { get; set; }
+  public float? offset_x { get; set; }
+  public float? offset_y { get; set; }
+  public float? rotation { get; set; }
```

Comment:
Can a CRS be considered valid if all it's properties are null? By adding nullability we're saying that it's ok for a CRS to not have a `wkt` (or any other field), is this correct?